exonum-java-binding-service-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -18,6 +18,8 @@
         <hamcrest.version>1.3</hamcrest.version>
         <mockito-core.version>2.22.0</mockito-core.version>
         <exonum-bom.version>0.2</exonum-bom.version>
+        <!--This property is set by jacoco and used in the surefire config further down-->
+        <argLine></argLine>
```

Comment:
Why do we set it in archetype — it's the template of a service, and there is no jacoco configuration (and I believe there must not be).